src/elements/index.d.ts

```diff
@@ -0,0 +1,3 @@
+const Elements: JSX.Element[];
```

Comment:
I guess just using an untyped intrinsic here?  Seems fine for now but we will want to type deeper if the dashboard starts converting to typescript.

Reply:
@ericpias in the new src/elements/index.ts file you can see that the Elements type is inferred from all the elements spread into the object